mne/_fiff/reference.py

```diff
@@ -74,6 +55,26 @@ def _check_before_reference(inst, ref_from, ref_to, ch_type):
     # Need to call setup_proj after changing the projs:
     inst._projector, _ = setup_proj(inst.info, add_eeg_ref=False, activate=False)
 
+
+def _check_before_reference(inst, ref_from, ref_to, ch_type):
```

Comment:
Can you move this back up? Moving it below `_check_ssp` will mess up `git blame` without any tangible benefit AFAICT?